nservicebus/operations/performance-counters_counters_core_[6].partial.md

```diff
@@ -0,0 +1,7 @@
+include: performance-counters-criticaltime
+
+include: performance-counters-note
+
+include: performance-counters-slaviolation
```

Comment:
It would cause us to add this text as a text for all 3 variations (3, 4-5 and 6). We can't move it up 1 level as then it would also be presented for Performance Counters 1.1 version in which case it is a bit redundant. I will remove a link for now